document/core/text/instructions.rst

```diff
@@ -113,7 +113,7 @@ All other control instruction are represented verbatim.
      \text{return} &\Rightarrow& \RETURN \\ &&|&
      \text{call}~~x{:}\Tfuncidx_I &\Rightarrow& \CALL~x \\ &&|&
      \text{call\_indirect}~~x{:}\Ttableidx~~y,I'{:}\Ttypeuse_I &\Rightarrow& \CALLINDIRECT~x~y
-       & (\iff I' = \{\}) \\
+       & (\iff I' = \{\ILOCALS~(\epsilon)^\ast\}) \\
```

Comment:
I think the corresponding note should also be updated, because this is not an empty context.

Reply:
Good point, done.